src/audio_switch.cpp

```diff
@@ -84,7 +84,7 @@ NxAudio::NxAudio() :
 	
 	mutexInit(&audio_mutex);
 
-	threadCreate(&audio_thread, switch_audio_thread, NULL, 0x10000, 0x2B, -2);
+	threadCreate(&audio_thread, switch_audio_thread, NULL, NULL, 0x10000, 0x2B, -2);
```

Comment:
Should use `nullptr`

Reply:
Yeah, should be used in other places as well. Just using it for consistency.